### DISCUSSION TITLE
Suggestion: use `latest` claude model by default?

### DISCUSSION THREAD

**Discussion by fredrikaverpil** [olimorris/codecompanion.nvim#368] [Ideas]
I noticed we have this today: https://github.com/olimorris/codecompanion.nvim/blob/be0e4f0534774bf4e53be7617e4e800cb39afcef/lua/codecompanion/adapters/anthropic.lua#L203

But according to the [docs](https://docs.anthropic.com/en/docs/about-claude/models#model-names), we can have `claude-3-5-sonnet-latest`.

WDYT?

**Comment by olimorris** [comment]
We added that in https://github.com/olimorris/codecompanion.nvim/pull/353, but bizarrely, it didn't support prompt caching.

I think it's best to manually test the latest models and add them in ourselves. It makes no sense to me that Anthropic would not have prompt caching setup on all new models.

**Reply by fredrikaverpil** [reply]
Wow, okay, yeah that was weird and I totally understand your reasoning. I'll close this. Thanks for your work on codecompanion!
